test/FastTests/Sharding/Client/ShardedHiloTests.cs

```diff
@@ -42,5 +40,122 @@ public void CanStoreWithoutId()
                 }
             }
         }
+
+        [RavenFact(RavenTestCategory.ClientApi | RavenTestCategory.Sharding)]
+        public async Task ReshardingOfHiloDoc()
```

Comment:
I think you need to add here `LicenseRequired` or move this to slow tests, otherwise the GH will fail here